src/main/java/com/rhombus/api/examples/userList.java

```diff
@@ -0,0 +1,182 @@
+package com.rhombus.api.examples;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.rhombus.ApiClient;
+import com.rhombus.sdk.UserWebserviceApi;
+import com.rhombus.sdk.domain.*;
+import org.apache.commons.cli.*;
+import org.apache.http.Header;
+import org.apache.http.message.BasicHeader;
+import org.glassfish.jersey.jackson.internal.jackson.jaxrs.json.JacksonJaxbJsonProvider;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLSession;
+import javax.ws.rs.client.ClientBuilder;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class userList
+{
+
+    private static ApiClient _apiClient;
+    private static UserWebserviceApi _userwebservice;
+
+    public static void main(String[] args) throws Exception
+    {
+        final Options options = new Options();
+        options.addRequiredOption("a", "apikey", true, "API Key");
+        options.addOption("c", "csv", true, "Name the csv file for the names");
+        options.addOption("r", "report", true, "Name the file folder that the csv will be in");
```

Comment:
Since we are only outputting a single csv report, no need to create a folder to put one file in. Just create the file in the current directory or use the full path if a user gives it in the `-r` argument.